x-pack/plugins/alerting/server/rules_client/tests/update.test.ts

```diff
@@ -1646,55 +1645,17 @@ describe('update()', () => {
         },
       });
 
-      expect(taskManager.runNow).not.toHaveBeenCalled();
+      expect(taskManager.bulkUpdateSchedules).not.toHaveBeenCalled();
     });
 
-    test('updating the alert should not wait for the rerun the task to complete', async () => {
```

Comment:
I believe we don't need this test anymore: we don't run task anymore, only doing REST API calls.
So, once user updates rule's schedule, there will be no time expensive run of rule, only task SO update.
In that case, we can wait to this update finished and we won't need to leave unfinished HTTP requests once return response of `RulesClient.update`